### ISSUE TITLE
Add succinct definition of sustainability to intro

### ISSUE THREAD

**Issue by TzviyaSiegman** [w3c/sustainableweb-wsg#191]
WCAG includes a definition of [Accessibility-supported](https://www.w3.org/TR/WCAG22/#dfn-accessibility-supporte), clearly identifying what must happen "To qualify as an accessibility-supported use of a web content technology". It would be helpful for us to write a short dfn of what we mean by Sustainable in this document.

## Proposal:
For the purposes of these guidelines, sustainability means a technology or policy that enables one or many of the following:
* Users or organizations to minimize their environmental impact (e.g. reduce CO2e, reduce water usage)
* Users or organizations to report on their environmental impact (e.g. reduce CO2e, reduce water usage)
* Users or organizations to improve policies around planetary, people, and prosperity principles

**Comment by codewordcreative** [comment]
Good - note that sustainability is also in [the list here](https://github.com/w3c/sustainableweb-wsg/issues/106). It was already on the agenda among words we would like to operationalise.

**Comment by systemstree** [comment]
We need to be careful of the use of the word regenerative. Can web ever be regenerative?

**Comment by systemstree** [comment]
@ines-akrap what if we also add the details of the note to the definition? Something like this: "An approach to prioritize people and the planet by designing, developing, and operating clean, efficient, open, honest, and resilient products and services within responsible environmental, technical, and human constraints." 

Also, can we use the word boundaries instead of constraints?

**Comment by codewordcreative** [comment]
Regarding existing ideas, a couple of thoughts/comments:
- We should aim for simplicity. Lists of words should be converted into bullets, but only if the longer list is needed.

I like Hidde's approach, but would want to tweak things a little:

Web Sustainability Guidelines:

    An approach to designing, developing, and operating digital products and services that puts people and the planet first: working responsibly within environmental, technical, and human constraints.

-- Approach is slightly off. Just minor tweaks...
Maybe 

> Guidance to support the designing, development, and operation of digital products and services that puts people and planet first, working responsibly within environmental, technical, and human constraints.

Since web sustainability isn't something that can be simply done and completed, perhaps "the ambition" or similar.

> Web Sustainability:
> The ambition to design, develop, and operate digital products and services such that they meet the needs of the present while ensuring future generations can meet their own needs.

**Comment by systemstree** [comment]
I like the definition that @codewordcreative has come up with:

>  The ambition to design, develop, and operate digital products and services such that they meet the needs of the present while ensuring future generations can meet their own needs.

Also, when we talk about "putting the people and the planet first", what does it mean? It is too broad. The above definition is much clearer and can easily be understood by many.

I would have preferred the following definition expanding on @codewordcreative 's definition to make it more specific to web sustainability, but it would be overly lengthy:

> The ambition to design, develop, and operate digital products and services such that they meet the needs of the present while ensuring future generations can continue to design, develop and operate digital products to meet their own needs.

**Comment by TzviyaSiegman** [comment]
I commented elsewhere 
I recommend moving the clear definition to the Intro - before the plain lang section, include a succinct definition of Web Sustainability. Consider moving https://w3c.github.io/sustainableweb-wsg/summary.html#what-is-web-sustainability section to the main document (and let's remove "the ambition" - wording in the summary doc is clear)

**Comment by TzviyaSiegman** [comment]
Proposal: Web Sustainability is an approach to designing, developing, and operating digital products and services that puts people and the planet first: working responsibly within environmental, technical, and human constraints.
A sustainable web experience aims to be:
* Clean: hosted using carbon-free or renewable energy;
* Efficient: using the fewest resources possible;
* Open: transparent, accessible, and user-controlled;
* Honest: avoiding manipulation, exploitation, or dark patterns;
* Regenerative: supporting communities and ecosystems;
* Resilient: designed to endure and adapt under changing circumstances.
(with references to Sustainable Web Manifesto and UN)

**Comment by systemstree** [comment]
What if we take @ines-akrap definition and include everything in a single sentence?

> An approach to designing, developing, and operating digital products and services that leads to a web that is efficient, open, honest, regenerative and resilient.

Other than "regenerative", most of the other values can be easily understood by many. But again, can web truly be regenerative?

**Comment by hidde** [comment]
@systemstree I like it in one sentence, but miss 'clean' in yours?

If we add this definition to spec, I'd suggest we add a note with the explanations for each of the aims. Like you say, each aim can be interpreted in many ways … so we need to explicitly document what we mean by it (I know I'll forget… but also… readers need to a way to find out what it means in order to apply it effectively).

**Comment by systemstree** [comment]
Thank you @hidde for noticing. Updating the definition here. I agree, we an explicit document specifying what each aim.

> An approach to designing, developing, and operating digital products and services that leads to a web that is clean, efficient, open, honest, regenerative and resilient.

Adding to @ines-akrap definition of regenerative, should we mention current and future generations:  "Regenerative: supporting current and future communities and ecosystems;"?

**Comment by airbr** [comment]
Interesting issue to read with a little time passing- 

As a supporter of the guidelines I am basically more interested in what core word choices I use to communicate to specific audiences. I notice how there is a logic tree in WCAG:
https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported

Basically how do we educate people its not a strict definition, its a logic tree of questions? (IF we follow that example as a indicator)

**Comment by AlexDawsonUK** [comment]
That's not the case in WSG though. It should be a **collective of all of the things defined** - as per the [Sustainable Web Manifesto](https://www.sustainablewebmanifesto.com/). This naturally applies to the whole document.

We shouldn't be asking people to pick and choose, else you'll end up, for example, having hosting providers proclaim they are sustainable when they suffer extensive downtime (not resilient), they don't use clean energy, just pay someone else to do it for them (RECs and offsets), they dump all their staff for an AI-first agenda (not regenerative), they use false-flagging of real humans as bots and inaccessible CAPTCHAs to burn more carbon and train AIs (not efficient)... but they're still open about all the awful stuff they're doing so they're compliant. And yes that's based on a real-world example!

At the criterion level, naturally only pieces will apply but for purposes of conformity, we hope people look at the whole.

**Comment by airbr** [comment]
Very good description of the big picture goals there! no attempt at a counter-argument at all here. Still I find that for dealing with some situations certain things get emphasized more as relevant to ones role. 

The logic tree stands out to me as a way to narrow and focus I admire that idea generally
